docs/core/integration-quality-scale/rules/iqs049.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "IQS049 - The documentation describes how data is updated"
+---
+
+IQS049 is as follows:
+
+The documentation describes how data is updated
+
+## Reasoning
+
+For the user to know how the integration works, we should describe how the data of the integration is updated.
+Because this will help users create an expectation on how well the integration works for their use case.
+A motion sensor that only polls every 5 minutes is less usable than a motion sensor that actively pushes updates.
+
+Since users can define their own polling interval for polling integrations, we should add at what rate we poll now and describe any limitations.
+For example if the device we connect to has known problems handling too many requests, we should describe that in the documentation.
+
+## Example implementation
+
+```markdown
+## Data updates
+
+My integration fetches data from the device every 5 minutes by default.
+Newer devices (the ones running MyOS) have the possibility to push data.
+At the start of the integration we try to enable that, and if it fails we fall back to polling.
```

Comment:
```suggestion
By default, my integration requests ("polls") data from the device every 5 minutes.
However, newer devices (the ones running MyOS) notify the integration whenever new data is available. This process is called "data pushing". Devices that push notifications instead of having the integration poll at regular intervals can improve performance and reduce unnecessary network traffic.
Whenever the integration is started, we try to enable pushing. If it fails, we fall back to ({% term polling %}).
```

Reply:
I do think that data pushing is something general and not specific to the integration, so I am not sure if we should explain this on every page

Reply:
I see your point...